README.md

```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
 #### Debian-based distro:
 
     sudo apt-get install nodejs git build-essential python2.7
+    #Raspberry Pi Cjdns Install
+
+##### Raspberry Pi
```

Comment:
I don't think sticking a huge section for Pi right in the middle of the README.md is a good idea. I suggest modifying https://github.com/tomeshnet/prototype-cjdns-pi2/blob/master/scripts/install into a scripted install and stick it in **contrib**, then add a line in the README to just run that